src/app/ui/dock.cpp

```diff
@@ -10,9 +10,19 @@
 
 #include "app/ui/dock.h"
 
+#include "app/app.h"
+#include "app/i18n/strings.h"
+#include "app/ini_file.h"
+#include "app/modules/gfx.h"
+#include "app/pref/preferences.h"
 #include "app/ui/dockable.h"
 #include "app/ui/skin/skin_theme.h"
+#include "layout_selector.h"
+#include "main_window.h"
```

Comment:
Please use the full path relative to `src/` folder to include files:
```suggestion
#include "app/ui/layout_selector.h"
#include "app/ui/main_window.h"
```